specs/carving.table

```diff
@@ -0,0 +1,17 @@
+table_name("carving")
```

Comment:
Maybe change name to carver? My reasoning is that when you initiate a carve it is most definitely a carver and selecting * returns information from the carver.